test/service/tests/runtime_upgrade.rs

```diff
@@ -25,6 +25,7 @@ use cumulus_test_runtime::GetUpgradeDetection;
 use sp_runtime::generic::BlockId;
 
 #[substrate_test_utils::test]
+#[ignore]
```

Comment:
any comment on this?

Reply:
https://github.com/paritytech/cumulus/issues/524